### ISSUE TITLE
Suggestion: Improve visibility of tags

### ISSUE THREAD

**Issue by henneysq** [fieldtrip/website#754]
## Issue

Currently, the tags associated with a tutorial are presented at the top of the page. Thus, as you scroll through a tutorial, the tags (which might be of interest to you later) move out of sight (and out of mind.

### Current view at the top of page:

![Screenshot 2024-01-23 at 14 19 29](https://github.com/fieldtrip/website/assets/120719655/24b81cb5-d1da-4529-b7e9-4e0dc1c77aed)

## Suggestion

I suggest moving (or adding) the tags as a fixed column on the left white margin (similar to the TOC on the right side). This would allow the reader to repeatedly re-reference the tags throughout the tutorial.

### Suggested view, after scrolling down the page:

![Screenshot 2024-01-23 at 14 15 28](https://github.com/fieldtrip/website/assets/120719655/79c8deee-be3a-409f-a716-4f6e9bbba4eb)

**Comment by robertoostenveld** [comment]
I don't think this would be very difficult to implement. It would involve moving the code from https://github.com/fieldtrip/website/blob/master/_includes/tags over to https://github.com/fieldtrip/website/blob/master/_includes/left, and making a (vertical) list rather than a horizontal line.
